CHANGELOG.md

```diff
@@ -701,6 +703,17 @@ filename | sha256 hash
 
 [Documentation](https://docs.k8s.io) & [Examples](https://releases.k8s.io/release-1.6/examples)
 
+## Known Issues for v1.6.5
+
+* If you use the [GLBC Ingress Controller](https://github.com/kubernetes/ingress/tree/master/controllers/gce),
```

Comment:
Since the issue only occurs when the following three are all meet:
* upgrade pre-1.6.4 to 1.6.4 and above
* use GLBC ingress controller
* Edited the default setting in GLBC Ingress controller

Can we make that more clear in the first sentence by including the last condition?